git/git.go

```diff
@@ -71,6 +71,50 @@ func UncommittedChangeCount() (int, error) {
 	return count, nil
 }
 
+type Commit struct {
+	Sha   string
+	Title string
+}
+
+func Commits(baseRef, headRef string) ([]*Commit, error) {
+	logCmd := GitCommand(
+		"log", "--pretty=format:%H,%s",
+		fmt.Sprintf("%s..%s", baseRef, headRef))
+	output, err := utils.PrepareCmd(logCmd).Output()
+	if err != nil {
+		return []*Commit{}, err
+	}
+
+	commits := []*Commit{}
+	sha := 0
+	title := 1
+	for _, line := range outputLines(output) {
+		split := strings.SplitN(line, ",", 2)
+		if len(split) != 2 {
+			continue
+		}
+		commits = append(commits, &Commit{
+			Sha:   split[sha],
+			Title: split[title],
+		})
+	}
+
+	if len(commits) == 0 {
+		return commits, fmt.Errorf("could not find any commits between %s and %s", baseRef, headRef)
+	}
+
+	return commits, nil
+}
+
+func CommitBody(sha string) (string, error) {
+	showCmd := GitCommand("show", "-s", "--pretty=format:%b", sha)
```

Comment:
Ditto `log.showSignature=false` would be a good idea https://github.com/github/hub/commit/70e63ece56714eb90a14e0ecf6b7d80b0511e0ab